### DISCUSSION TITLE
Showing html in toast (disable escape)

### DISCUSSION THREAD

**Discussion by puschie286** [primefaces/community#955] [PrimeNG] [answered]
Hi,
would be great if the toast component have a "escape" parameter / input (like message does) to show html content

**Comment by cetincakiroglu** [comment, marked as answer]
You can use headless template https://primeng.org/toast#headless

**Reply by puschie286** [reply]
that’s the workaround we use - but you need to copy almost the whole component only to be able have the same functionality like escape parameter in message

**Reply by cetincakiroglu** [reply]
We'll wait for community upvotes to implement this feature.

**Comment by zJaaal** [comment]
This would be a great feature. It would match the functionality on the Message component (which uses the same service) and it's not doable to create a whole template just to add styling to the text on a detail.
